docs/en/latest/plugins/forward-auth.md

```diff
@@ -227,8 +228,8 @@ curl -X PUT 'http://127.0.0.1:9180/apisix/admin/routes/1' \
 Send a POST request with the `tenant_id` header:
 
 ```shell
-curl -i http://127.0.0.1:9080/post -X POST -d '{
-   "tenant_id": 123
+curl -i http://127.0.0.1:9080/post -H "Content-Type: application/json" -X POST -d '{
+   "tenant_id": "123"
```

Comment:
If user accidentally use the number instead of the string, they will run into a 500 error.

<img width="722" height="337" alt="image" src="https://github.com/user-attachments/assets/8d5f6ab0-019d-4db1-b304-b19164904e14" />

It is not entirely clear based on the current example why (as a user) I have to use string. I think this error should be caught, but I'll leave the judgement to you.